.github/workflows/ui-rm-upgrade-proxy.yaml

```diff
@@ -0,0 +1,64 @@
+# This workflow calls the master E2E workflow with custom variables
+name: UI-RM_Upgrade_Proxy
+run-name: ${{ github.event_name == 'workflow_dispatch' && format('`{0}` on `{1}` - `{2}` destroy={3}', inputs.rancher_version, inputs.upstream_cluster_version, inputs.grep_test_by_tag, inputs.destroy_runner) || github.workflow }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      qase_run_id:
+        description: Qase run ID where the results will be reported. (Please update with custom run ID.)
+        default: none
+        type: string
+      destroy_runner:
+        description: Destroy the auto-generated self-hosted runner
+        default: true
+        type: boolean
+      rancher_version:
+        description: Rancher version channel/version/head_version latest/2.9.x[-rc1], prime/2.9.x, prime/devel/2.9, alpha/2.x.x-alphaX, prime-optimus-alpha/2.x.x-alphaX
+        default: latest/devel/2.9
+        type: string
+        required: true
+      upstream_cluster_version:
+        description: K3s upstream cluster version where to install Rancher
+        default: 'v1.28.8+k3s1'
+        type: string
+        required: true
+      proxy:
+        description: Creates proxy server and pass proxy server address to Rancher.
+        default: rancher
+        type: string
```

Comment:
I would use boolean instead, string `rancher` makes no sense to me.

Reply:
Maybe I am wrong. But I recall a string was needed for the way the upgrade script was done.